microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -153,6 +155,8 @@ chrome.tabs.onUpdated.addListener(async (tabId, info, tab) => {
 
 When a user switches to a tab or site for which the sidebar is not enabled, the sidebar is hidden.
 
+Note: Currently, the sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.<!-- todo: if behavior changes, update or remove statement -->
```

Comment:
should we have same note here as well as above?  

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-3328#enable-a-sidebar-for-a-specific-site-only - below the code listing, find "not automatically displayed"

Reply:
This is a good idea.